cmd/cli/main.go

```diff
@@ -566,6 +573,25 @@ func identityUpdatesStress(logger *zap.Logger, options *CLI) {
 	}
 }
 
+func startChainWatcher(logger *zap.Logger, options *CLI) {
+	ctx := context.Background()
```

Comment:
does this react to `cmd-c` correctly? I am never certain how contexts vs signals interact